packages/big-design/src/components/MultiSelect/spec.tsx

```diff
@@ -369,6 +373,29 @@ test('clicking on disabled select options should not trigger onItemClick', async
   await waitForElement(() => screen.getByRole('option', { name: /mex/i }));
 });
 
+test('opening the MultiSelect triggers onOpen', async () => {
+  const { getAllByRole, queryByRole } = render(MultiSelectMock);
+  const button = getAllByRole('button')[2];
+
+  fireEvent.click(button);
+
+  expect(onOpen).toHaveBeenCalled();
+
+  await waitForElement(() => queryByRole('listbox'));
+});
+
+test('closing the MultiSelect triggers onClose', async () => {
+  const { getAllByRole, queryByRole } = render(MultiSelectMock);
+  const button = getAllByRole('button')[2];
```

Comment:
It is better to target the actual button as opposed to use `[2]`. Not sure if it is feasible in this case, but it is better to use `getByRole('button', { name: /text of the button/i })`